peachjam_search/serializers.py

```diff
@@ -105,10 +105,12 @@ def get_nature(self, obj):
         return obj["nature" + self.language_suffix]
 
     def get_outcome(self, obj):
-        val = obj["outcome" + self.language_suffix]
-        if val is not None:
-            val = list(val)
-        return val
+        if hasattr(obj, "outcome" + self.language_suffix):
```

Comment:
Why would this have to be different to the ones above?

Reply:
The migration for mapping is not actually updating the indexed cocument object, only the mapping object. This causes a key error in the results when searching. I'm starting to think that we may have to rebuild the index :( 
Even when resaving a document, the indexed document is not picking up the new field. only rebuilding the index fully works. I may be missing something. How did you get provisions to work?

Reply:
makes sense